### ISSUE TITLE
Does not work on Visual Studio 2015

### ISSUE THREAD

**Issue by conficient** [landofjoe/NuspecPackager#1]
I have a clean Windows 10 installation with Visual Studio 2015. When I click `Package .nuspec` on a nuspec file in my solution, I get the following error:

```
Could not load file or assembly 'Microsoft.VisualStudio.Services.Integration, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
```

This library seems to be part of the Visual Studio SDK in versions 2013 and earlier, but seems to have been removed from VS2015 SDK. Since it's only purpose was to call OutputWindowHelper I assume it might be possible to amend this.

**Comment by conficient** [comment]
I forked and branched this to create a simple fix which uses `Trace` - I don't know if this is the best alternative so I won't submit a pull request. However it compiled, installed and ran under VS2015.

https://github.com/conficient/NuspecPackager/commit/1861817b21821d98404e6a117d9356cf22a58ac1#diff-a7b16c26a7ab845a66358fb80446cac9

**Comment by conficient** [comment]
See new commit on my VS2015 branch for a fix that creates its own output window 

https://github.com/conficient/NuspecPackager/commit/00149631070f01651f0fce8d1af9ac9169a5bc86

**Comment by funky-jojo** [comment]
Thanks for the update and for the custom output window.  Much better.  I didn't see a pull request, so I have implemented conficient@0014963 back into this repo with a few tweaks to clean up option processing and improve output to indicate overall success.
